airflow/ui/src/pages/DagsList/Dag/Tabs.tsx

```diff
@@ -24,23 +24,26 @@ import {
   useSearchParams,
 } from "react-router-dom";
 
-import type { DAGResponse } from "openapi/requests/types.gen";
+import type { DAGDetailsResponse } from "openapi/requests/types.gen";
 import { DagIcon } from "src/assets/DagIcon";
 import { capitalize } from "src/utils";
 
+import { DagDocumentation } from "./DagDocumentation";
 import { DagVizModal } from "./DagVizModal";
 
 const tabs = ["runs", "tasks", "events", "code"];
 
 const MODAL = "modal";
 
-export const DagTabs = ({ dag }: { readonly dag?: DAGResponse }) => {
+export const DagTabs = ({ dag }: { readonly dag?: DAGDetailsResponse }) => {
   const { dagId } = useParams();
   const [searchParams, setSearchParams] = useSearchParams();
 
   const modal = searchParams.get(MODAL);
 
   const isGraphOpen = modal === "graph";
+  const isDocsOpen = modal === "docs";
```

Comment:
I think dag docs can use the regular `useState` instead of url parameters. Graph is special because it's full page.

Reply:
Thanks. Used `useState` from `Dag` and passed it to `Header` and `Tabs` where the state is updated .